<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchByNameDialog</class>
 <widget class="QDialog" name="SearchByNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>682</width>
    <height>468</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>682</width>
    <height>468</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="layerList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>391</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Which layers to search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>What to search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchString">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="searchResult">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SearchByNameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search</sender>
   <signal>clicked()</signal>
   <receiver>SearchByNameDialog</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchResult</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>SearchByNameDialog</receiver>
   <slot>showGISObject(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
  <slot>showGISObject(QListWidgetItem*)</slot>
 </slots>
</ui>
